### PR TITLE
Feature/game loop with review+Night Selection

### PR DESCRIPTION
**Pull request by SabrinaTheCookie**

### CODE REVIEW

File: Assets/GameManager.cs
Comment (JamesCoburn246) [inline Assets/GameManager.cs:1]: This isn't in /_Scripts/ and seems empty?

File: Assets/NightSelection.cs
Comment (JamesCoburn246) [inline Assets/NightSelection.cs:1]: Not in /_Scripts/?

File: Assets/_Scripts/StageManager.cs [hunk 1]
```diff
@@ -145,7 +150,7 @@ private void CheckIsFull(StagePlacement placement)
         {
             //If an empty spot (and turned on!), then its not full.
             (bool, bool) pointStatus = placementPoint.IsOccupied();
-            if (pointStatus is {Item1: false, Item2: false} && placementPoint.gameObject.activeSelf)
+            if ((!pointStatus.Item1 || !pointStatus.Item2) && placementPoint.gameObject.activeSelf)
```
Comment (JamesCoburn246) [inline Assets/_Scripts/StageManager.cs:153]: This one-liner might be able to be simplified?

**Review by JamesCoburn246 — CHANGES_REQUESTED** [review]

File: Assets/GameManager.cs
Comment (SabrinaTheCookie) [reply Assets/GameManager.cs:1]: Fixed

File: Assets/NightSelection.cs
Comment (SabrinaTheCookie) [reply Assets/NightSelection.cs:1]: Fixed

File: Assets/_Scripts/StageManager.cs (hunk 1 above)
Comment (SabrinaTheCookie) [reply Assets/_Scripts/StageManager.cs:155]: 🤷 just ship it.

Good enough for a jam

**Review by JamesCoburn246 — APPROVED** [review]

**Pull request merged**